packages/angular/src/variables.scss

```diff
@@ -4,7 +4,7 @@
   }
 
   .dx-theme-material {
-    --change-password-popup-height: 260px;
+    --change-password-popup-height: auto;
```

Comment:
It's a bit odd for me to see auto used in a css variable, hovewer functionally it seems allright